### PR TITLE
Cats effect testing does not support munit

### PR DESCRIPTION
**Pull request by Guisanpea**
This PR fixes the fact that the documentation points to it

### CODE REVIEW

**Review by vasilmkd — APPROVED** [review]
Thanks for the contribution!

File: docs/guides/testing.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ title: Testing
 
 Relatively few libraries support cats-effect directly at this time. However, most (if not all) popular testing frameworks have libraries offering some level of integration
 
-- [cats-effect-testing](https://github.com/djspiewak/cats-effect-testing): Supports Scalatest, Specs2, munit, minitest, µTest, and scalacheck
+- [cats-effect-testing](https://github.com/djspiewak/cats-effect-testing): Supports Scalatest, Specs2, minitest, µTest, and scalacheck
```
Comment (sideeffffect) [inline docs/guides/testing.md:12]: It might as well point to https://github.com/typelevel/cats-effect-testing

Comment (vasilmkd) [reply docs/guides/testing.md:12]: That's a good observation, thanks. I do believe that the URL redirects correctly, but the change would be appreciated.

Comment (sideeffffect) [inline docs/guides/testing.md:12]: It's https://github.com/typelevel/munit-cats-effect what supports MUnit

**Review by sideeffffect — CHANGES_REQUESTED** [review]

**Pull request merged**